terraform/modules/happy-github-ci-role/eks.tf

```diff
@@ -0,0 +1,70 @@
+//eks:DescribeCluster on resource: arn:aws:eks:us-west-2:401986845158:cluster/si-playground-eks-v2
+/*{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": "eks:DescribeCluster",
+            "Resource": "arn:aws:eks:us-west-2:401986845158:cluster/si-playground-eks-v2"
+        }
+    ]
+}*/
+
+data "aws_iam_policy_document" "eks" {
+  statement {
+    sid = "GhActionsDescribeCluster"
+    actions = [
+      "eks:DescribeCluster",
+    ]
+    resources = [
+      var.eks_cluster_arn,
+    ]
+  }
+}
+resource "aws_iam_role_policy" "eks" {
+  name   = "gh_actions_eks_describe_cluster_${local.namespace}"
+  policy = data.aws_iam_policy_document.eks.json
+  role   = local.role_name
+
+  depends_on = [module.gh_actions_role]
+}
+
+//[FATAL]: unable to initialize the happy client: unable to retrieve integration secret: Unauthorized
+// PATCH ADD the following to aws-config-map
+/*- "groups":
+  - "system:masters"
+  "rolearn": "arn:aws:iam::401986845158:role/gh_actions_si_rdev_happy_eks_rdev_happy"
+  "username": "gh_actions_si_rdev_happy_eks_rdev_happy"
+  */
+// TODO still not working exactly
+data "kubernetes_config_map" "aws-auth" {
```

Comment:
@alexlokshin-czi I'm still playing with this, but I think we can do something like to patch the aws-auth configmap after it is created and then add the CI roles automatically whenever they are created